vacuum-advisor/vacuum-simulator.mdx

```diff
@@ -0,0 +1,182 @@
+  The current simulation can't simulate this situation well, therefore dead rows
+  simply keep increasing with UPDATEs and trigger autovacuum earlier.
+* The avoidable growth graph may over report avoidable growth size due to the
+  limitation of the simulation.
+  You can check the Estimated Table Bloat panel of Schema Statistics
+  VACUUM/ANALYZE Activity page for the current estimated table bloat to access
+  the actual impact.
+
+
+## Triggered by freeze age
+
+To prevent transaction ID wraparound, Postgres runs autovacuums periodically to
+freeze older rows. We also say this as "freeze old transaction IDs",
+You can learn more about freezing in the
+[What is freezing?](/docs/vacuum-advisor/what-is-freezing) page.
+
+![Triggered by freeze age](vacuum_simulator_freezing.png)
+
+When either the age of the oldest unfrozen transaction ID or the oldest unfrozen
+multixact ID exceeds `autovacuum_freeze_max_age` or
+`autovacuum_multixact_freeze_max_age` respectively, freeze age autovacuums will
+be triggered. These autovacuums are called anti-wraparound autovacuums.
+
+With VACUUM Simulator, you can tweak `autovacuum_freeze_max_age` and
+`autovacuum_multixact_freeze_max_age` in this section.
+`autovacuum_multixact_freeze_max_age` can be only tweaked if multixact ID is
+used. Lowering these configuration settings triggers autovacuums more, and
+increasing these configuration settings will decrease autovacuums triggered by
+freeze age.
+
+### Limitation
+
+* The simulation is using the data of average transactions (or multixacts) per
+  second (tps).
+  If the tps varies a lot depending on the time of the day or day of the week,
+  the simulation may look a lot different than the actual data.
+* Autovacuums triggered by other causes usually try to freeze old rows.
+  The simulation also freezes them, however, there are cases that freezing won't
+  happen when there are nothing can be frozen, or autovacuum is more focused on
+  cleaning up dead rows.
+  This can cause the transaction age to be reset even when it shouldn't be in
+  the simulation, potentially cause the simulation to trigger less autovacuums
+  by freeze age.
+
+
+## Triggered by inserts
+
+Inserts autovacuum is introduced on Postgres 13.
+With an inserts only table, because it won't generate any dead rows, autovacuums
+won't be triggered by dead rows. It can still be triggered by freeze age, but
+depending on the database workload, it can be a long time until it's triggered.
+Inserts autovacuum is triggered to freeze old transaction IDs like freeze age
+autovacuum, but more importantly, it is triggered to update a visibility map.
+Each table has a visibility map to keep track of which pages contain only tuples
+that are known to be visible to all active transactions. For the case like an
+inserts only table, all the pages of the table should be visible here and less
+likely to be not visible moving forward (will be flagged as not visible if there
+is any write operation on that page).
```

Comment:
```suggestion
that are known to be visible to all active transactions. For a case like an
append-only table, all pages of the table should be all-visible, and are
likely to stay all-visible moving forward (they will be flagged as not all-visible if there
is any write operation on that page).
```
I think it's important to talk about "all-visible" specifically, not just "visible", to keep things clear.